Yona/.rswiftignore

```diff
@@ -0,0 +1,4 @@
+# Fabric and Crashlytics files
+fabric.apikey
```

Comment:
These three files are removed, so you can remove this ``.rswitfignore` file and all references to these three files (several more references exist).